src/ScheduleService.Validation/Workspace/PatchWorkspaceRequestValidator.cs

```diff
@@ -0,0 +1,19 @@
+﻿using FluentValidation;
+using FluentValidation.Results;
+using LT.DigitalOffice.ScheduleService.Models.Dto.Requests.Workspace;
+using LT.DigitalOffice.ScheduleService.Validation.Workspace.Interfaces;
+using System.Threading;
+using System.Threading.Tasks;
+
+public class PatchWorkspaceRequestValidator : AbstractValidator<PatchWorkspaceRequestValidator>, IPatchWorkspaceRequestValidator
+{
+  public ValidationResult Validate(PatchWorkspaceRequest instance)
+  {
+    throw new System.NotImplementedException();
+  }
+
+  public Task<ValidationResult> ValidateAsync(PatchWorkspaceRequest instance, CancellationToken cancellation = default)
```

Comment:
Why do you need 2 methods?

Reply:
This is how the IDE proposes to implement the IValidator interface. This is a temporary solution, the project will not be build without it.

Reply:
What does T mean in AbstractValidator?

Reply:
You can see here https://github.com/LT-Students/DigitalOffice-UserService/blob/develop/src/UserService.Validation/User/CreateUserRequestValidator.cs